### PR TITLE
add to files

### PR DESCRIPTION
**Pull request by ChaikaS**

### CODE REVIEW

File: HTML, CSS task(gulp)/#src/scss/1.header.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.wrapper {
```
Comment (mikalaider) [inline HTML, CSS task(gulp)/#src/scss/1.header.scss:1]: Please make the correct order for all css properties. This is the best practice. Here is an article that may help you:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

File: HTML, CSS task(gulp)/#src/_main.html [hunk 2]
```diff
@@ -0,0 +1,268 @@
+<main class="main">
+    <div id="my__modal">
```
Comment (mikalaider) [inline HTML, CSS task(gulp)/#src/_main.html]: `my__modal` is a bad name. What does it mean?

File: HTML, CSS task(gulp)/#src/_main.html [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+      <div id="modal__block"
+      >
+      <img id="img01"src="" alt="" />
+        <div id="caption"></div>
+      </div>
+    </div>
+  <section class="main__creative-agency">
+    <div class="main__creative-agency-were">we're</div>
+    <div class="main__creative-agency-h1"><h2>creative agency</h2></div>
+    <div class="main__creative-agency-text">
```
Comment (mikalaider) [inline HTML, CSS task(gulp)/#src/_main.html]: That's a bad name. Don't use a tag-specific substring like`h1'.

File: HTML, CSS task(gulp)/#src/_main.html [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+      <div class="main__history-of-agency-info-text">
+        Porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur,
+        adipisci velit, sed quia non numquam eius modi tempora incidunt ut
+        labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima
+        veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam,
+        nisi ut aliquid ex ea commodi modi tempora incidunt ut labore.
+      </div>
+      <button
+        type="button"
+        onclick="window.location.href='https://ru.wikipedia.org/wiki/%D0%98%D1%81%D1%82%D0%BE%D1%80%D0%B8%D1%8F_%D0%91%D0%B5%D0%BB%D0%BE%D1%80%D1%83%D1%81%D1%81%D0%B8%D0%B8'"
```
Comment (mikalaider) [inline HTML, CSS task(gulp)/#src/_main.html]: Bad idea to use inline JS

File: HTML, CSS task(gulp)/#src/scss/1.header.scss (hunk 1 above)
Comment (ChaikaS) [reply HTML, CSS task(gulp)/#src/scss/1.header.scss:1]: done

File: HTML, CSS task(gulp)/#src/_main.html (hunk 2 above)
Comment (ChaikaS) [reply HTML, CSS task(gulp)/#src/_main.html]: done

File: HTML, CSS task(gulp)/#src/_main.html (hunk 3 above)
Comment (ChaikaS) [reply HTML, CSS task(gulp)/#src/_main.html]: done

File: HTML, CSS task(gulp)/#src/_main.html (hunk 4 above)
Comment (ChaikaS) [reply HTML, CSS task(gulp)/#src/_main.html]: done

File: HTML, CSS task(gulp)/#src/_main.html (hunk 3 above)
Comment (ChaikaS) [reply HTML, CSS task(gulp)/#src/_main.html]: done

**Pull request merged**